src/moviesPanel/MovieCard.jsx

```diff
@@ -5,44 +5,39 @@ import ContextMenu from '../modal/ContextMenu.jsx';
 import MovieForm from '../modal/MovieForm';
 import MovieCardContent from './MovieCardContent';
 
-const MovieCard = ({ title, imageName, genre, releaseDate }) => {
+const MovieCard = ({ movie, onMovieClick }) => {
   const [openContextMenu, setOpenContextMenu] = useState(false);
   const movieForm = (
-    <MovieForm movieTitle={title} releaseDate={releaseDate} genre={genre} />
+    <MovieForm
+      movieTitle={movie.title}
+      releaseDate={movie.releaseDate}
+      genre={movie.genre}
+    />
   );
   const handleContextMenuClose = useCallback(
     () => setOpenContextMenu(false),
     []
   );
   return (
-    <div className='movieCard'>
-      <MovieCardContent
-        title={title}
-        imageName={imageName}
-        genre={genre}
-        releaseDate={releaseDate}
-      />
+    <div className='movieCard' onClick={() => onMovieClick(movie)}>
```

Comment:
useCallback

Reply:
used useCallback in App.jsx to create onMovieClick